libs/search-widget/package.json

```diff
@@ -1,4 +1,33 @@
 {
   "name": "@nuclia/widget",
-  "version": "0.0.1"
+  "version": "1.0.0",
+  "description": "Nuclia search widget",
+  "module": "nuclia.mjs",
+  "main": "nuclia.umd.js",
+  "license": "MIT",
+  "keywords": [
+    "Nuclia",
+    "NucliaDB",
+    "AI search"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/nuclia/frontend"
+  },
+  "bugs": {
+    "url": "https://github.com/nuclia/frontend/issues"
+  },
+  "peerDependencies": {
+    "@nuclia/core": "^1.1.3",
```

Comment:
you bumped the version the 1.1.4, so I guess we could bump this as well.

Reply:
^1.1.3 will get the latest 1.*.* anyway (thanks to `^`)